example_template.md

```diff
@@ -41,76 +44,119 @@ git clone https://github.com/jina-ai/examples
+### 🔎 Step 4: Query your data
+Next, we will deploy our query Flow.
 
-### Step 1: Index your data
+*Here describe the Query Flow. Be as specific as possible in describing how this Query Flow works and what is its input. You are encouraged to use code snippets, images, or whatever helps to clarify.*
 
-Describe the Index Flow. Be as specific as possible. You are encouraged to user code snippets, images, or whatever helps to clarify.
+Run the query Flow in your terminal like this:
 
-### Step 2: Search your data
+```
+python app.py search (replace as necessary)
+``` 
+______
 
-Describe the Query Flow. Be as specific as possible. You are encouraged to user code snippets, images, or whatever helps to clarify.
+## 📉 Understanding your results
+*Here include a short description of the results and how to interpret them if needed.*
 
 ## 🌀 Flow diagram
+This diagram provides a visual representation of the Flows in this example; Showing which executors are used in which order.
 
-Show the Flow for this example.
+*Here Show the Flow for this example.*
 
-## 🌟 Results
+## 📖 Optional: Extra information useful for the user
 
-Short description of the results and how to interpret them if needed.
+*Use this section to add extra information you think the user could benefit from.
+QueryLanguage, Faiss, Annoy for example.*
 
-## 🧞 Optional: Extra information useful for the user
+## 🔮 Overview of the files
+
+*Add a list with all folders/files in the example:*
+
+|                      |                                                                                                                  |
+| -------------------- | ---------------------------------------------------------------------------------------------------------------- |
+| 📂 `flows/`          | Folder to store Flow configuration                                                                               |
+| --- 📃 `index.yml`     | YAML file to configure indexing Flow                                                                             |
+| --- 📃 `query.yml`     | YAML file to configure querying Flow                                                                             |
+| 📂 `pods/`           | Folder to store Pod configuration                                                                                |
+| --- 📃 `encoder.yml`   | YAML file to configure encoder Pod                                                                               |
+| 📂 `workspace/`      | Folder to store indexed files (embeddings and documents). Automatically created after the first indexing   |
 
-You can use this section to add extra information you think the user could benefit from.
-QueryLanguage, Faiss, Annoy for example. 
+_____
 
-## 💫 Deploy with Docker
+## 🐋 Deploy with Docker
+To make it easier for you, we have built and published the Docker image for this example.
 
-Pre requirements:
+### ☑️ Pre-requirements:
```

Comment:
"Pre-requirements" isn't really a word. "Prerequisites" is, but it's not so well-known to non-native speakers.

Let's just change all instances of this to "Requirements" (We can drop the pre-)